README.md

```diff
@@ -27,8 +27,10 @@ Heavily "inspired" by the
 - [Coveralls](https://coveralls.io/github/cisagov) - Test coverage tracking
 - [GitHub Actions](https://github.com/features/actions) -
   Continuous integration and delivery
-- [LGTM](https://lgtm.com/search?q=cisagov) - Code analysis platform for
-  finding zero-days and preventing critical vulnerabilities
+- (Deprecated) [LGTM](https://lgtm.com/search?q=cisagov) - Code analysis
+  platform for finding zero-days and preventing critical vulnerabilities
```

Comment:
I think this can be removed.
```suggestion
```

Reply:
I was going back and forth between removing the references now or waiting until we were completely done.  I'm fine either way, so will defer to your recommendation.